.travis.yml

```diff
@@ -1,13 +1,23 @@
 sudo: false
 language: scala
+services:
+- redis-server
 scala:
 - 2.11
 jdk:
 - oraclejdk8
 env:
   global:
-  - BRIDGE_USER=travis
+  - BRIDGE_USER=heroku
   - BRIDGE_ENV=dev
+  - WEBSERVICES_URL="https://webservices-develop.sagebridge.org"
+  - HOST_POSTFIX="-develop.sagebridge.org"
+  - UDD_SQS_QUEUE_URL="https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-WorkerPlatform-Request-dev"
+  - UPLOAD_CMS_CERT_BUCKET="org-sagebridge-upload-cms-cert-develop"
+  - CONSENTS_BUCKET="org-sagebridge-consents-develop"
+  - UPLOAD_BUCKET="org-sagebridge-upload-develop"
+  - UPLOAD_CMS_PRIV_BUCKET="org-sagebridge-upload-cms-priv-develop"
+  - ATTACHMENT_BUCKET="org-sagebridge-attachment-develop"
   - secure: SsM07kjCSWjGTFDhkCtTuZdM5WrChFXgRmeMMRJr4JlbtJoaaVUXcXDofZ+4NrESiOvW7CiRXK6Oz3I9eL7vn8RCwlN5jf2OlmD4vKIiiINlvuuEe6hEu738RO7nUGN8SFh2k4ryzJ6XldghjBiBvBEqzVhwXteLhF3AVCsRB1s=
```

Comment:
I'm wondering if we need to add the following VARs since they are set to different values in bridgepf-dev and bridgepf-uat on heroku:

BRIDGE_HEALTHCODE_KEY
BRIDGE_HEALTHCODE_REDIS_KEY
AWS_KEY_CONSENTS
AWS_SECRET_KEY_CONSENTS
SNS_KEY
SNS_SECRET_KEY